docs/quickstart/multisig-utxos-with-avalanchejs.md

```diff
@@ -0,0 +1,1145 @@
+
+If an account A wants to send 10 AVAX to account B, then it has to include all those unspent outputs in a transaction, that are owned by A and whose sum of amounts in those outputs is more than or equal to 10. These UTXOs will be included as inputs in a transaction. Account A also has to create outputs with amount 10 and the owner being the receiver (here B). There could be multiple outputs in the outputs array. This means, that using these UTXOs, we can create multiple outputs with different amounts to different addresses.
+
+Once the transaction is committed, the UTXOs in the inputs will be consumed and outputs will become new UTXOs for the receiver. If the inputs have more amount unlocked than being consumed by the outputs, then the excess amount will be burned as fees. Therefore, we should also create a change output which will be assigned to us, if there is an excess amount in the input.
+
+![](/img/multisig-utxos-1.png)
+
+
+## Multi-Signature UTXOs
+
+UTXOs can be associated with multiple addresses. If there are multiple owners of a UTXO, then we must note the `threshold` value. We have to include signatures of a threshold number of UTXO owners with the unsigned transaction to consume UTXOs present in the inputs. The threshold value of a UTXO is set while issuing the transaction.
+
+We can use these multi-sig UTXOs as inputs for multiple purposes and not only for sending assets. For example, we can use them to create subnets, add delegators, add validators, etc.
+
+## Atomic Transactions
+
+On Avalanche, we can even create cross-chain outputs. This means that we can do a native cross-chain transfer of assets. These are made possible through **Atomic Transactions**. This is a 2-step process -
+* Export transaction on source chain
+* Import transactions on the destination chain
+
+Atomic transactions are similar to other transactions. We use UTXOs of the source chain as inputs and create outputs owned by destination chain addresses. When the export transactions are issued, the newly created UTXOs lie in the **Exported Atomic Memory**. These are neither on the source chain nor on the destination chain. These UTXOs can only be used as inputs by their owners on the destination chain while making import transactions. Using these UTXOs on the atomic memory, we can create multiple outputs with different amounts or addresses.
+
+![](/img/multisig-utxos-2.png)
+
+## UTXOs on C-Chain
+
+We can't use UTXOs on C-Chain to do regular transactions because it follows the account-based approach. In C-Chain, each address (account) is mapped with its balance, and the assets are transferred simply by adding and subtracting from this balance using the virtual machine.
```

Comment:
```suggestion
We can't use UTXOs on C-Chain to do regular transactions because C-Chain follows the account-based approach. In C-Chain, each address (account) is mapped with its balance, and the assets are transferred simply by adding and subtracting from this balance using the virtual machine.
```